roles/clevis/README.rst

```diff
@@ -0,0 +1,37 @@
+An ansible role to install clevis library for network bound disk encryption.
+
+**Role Variables**
+
+.. zuul:rolevar:: clevis_packages
+   :default: []
+
+   the clevis required installation packages
```

Comment:
Please do not indent these comments. Start each with a capital letter.

Reply:
Correct with captial letter